catalog/plugin/catalog-plugin-metacardbackup/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/MetacardBackupPlugin.java

```diff
@@ -96,28 +94,29 @@
             return processRequest;
         }
 
-        if (StringUtils.isEmpty(outputDirectory)) {
+        if (CollectionUtils.isEmpty(storageBackupPlugins)) {
             throw new PluginExecutionException(
-                    "Unable to delete backup ingested metacard; no output directory specified.");
+                    "Unable to delete backup ingested metacard; no metacard backup storage provider configured.");
         }
 
         List<ProcessDeleteItem> processUpdateItems = processRequest.getProcessItems();
         for (ProcessDeleteItem processUpdateItem : processUpdateItems) {
             Metacard metacard = processUpdateItem.getMetacard();
-            deleteBackupIfPresent(metacard.getId());
+
+            for (MetacardBackupStorageProvider storageProvider : storageBackupPlugins) {
+                try {
+                    storageProvider.deleteData(metacard.getId());
+                } catch (IOException | MetacardBackupException e) {
+                    LOGGER.debug("Unable to delete backed up metacard data for metacard: {}",
```

Comment:
It would be valuable to know which storage provider failed here, in the case where you have multiples. They're `Describable` now, so it's easy to get the name.